### ISSUE TITLE
Add a config value for page string max length

### ISSUE THREAD

**Issue by Mumberthrax** [The-Fireplace-Minecraft-Mods/Textbook#11]
**Is your feature request related to a problem? Please describe.**
Same issue as #9  except I believe I discovered it may only be string length and not byte amount being restricted. The new textbook update (2.2.0) with the byte restriction feature is extremely useful and works for most of my use cases - and I believe I was mistaken in identifying bytes as the issue instead of simple string length.

**Describe the solution you'd like**
in MinecraftBookConstraints.java, fitsOnPage() should check for a config value to compare string length to, instead of or in addition to the current comparison of 1024.

**Additional context**
I installed the [Stendhal mod by NebSpacefarer](https://modrinth.com/mod/stendhal) which shows character count in the in-game gui for each page. I was experimenting with the byte restriction feature of Textbook 2.2.0 to figure out what limit was set on a default paperMC server, and realized that none of the pages were exceeding 320 characters. I set the textbook byte restriction to 833 which seemed to work with a text i was using, and then tried importing a different one - an ascii art  - that had been severely affected by the papermc restrictions previously. Even though textbook was importing it without the byte restrictions affecting it, it had around 400 characters. Closing the gui it was edited by the server down to 320 characters.

[the ascii art page](https://gist.github.com/Mumberthrax/2238bc29a97b2b9d840c54ea82f8a72e)
![image](https://user-images.githubusercontent.com/4822803/147866463-4df0f9ec-3234-4324-b776-fb823aa0b598.png)

After papermc updates the page:
![image](https://user-images.githubusercontent.com/4822803/147866513-7f6df2d2-6518-4d39-8050-ceed5758a71a.png)

edit: [A modified version of the ascii art which is not edited by the server, in case it is useful](https://gist.github.com/Mumberthrax/86be418a851bb3212f34e60fb79fdb50)

[My current test text with plaintext characters](https://gist.github.com/Mumberthrax/6b12a2b293c74ded17bfdac2a66157dc)

[The same test text with fullwidth characters](https://gist.github.com/Mumberthrax/726006a6c71dc40089e6f1d728dfc8cb)

This proposed change is not essential for my purposes personally, but might be nice-to-have since it addresses the paperMC restrictions seemingly more directly. Sorry for my mistake on the matter before.

**Comment by The-Fireplace** [comment]
I don't believe your previous issue was a mistake, just a separate restriction that Paper has. I can see about making the max characters per page configurable at some point.
